client/components/thank-you-v2/footer/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, Gridicon } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { isOutsideCalypso } from 'calypso/lib/url';
+
+import './style.scss';
+
+export type ThankYouDetailProps = {
+	title?: string;
+	description?: string;
+	buttonText?: string;
+	href?: string;
+	clickEventName?: string;
+};
+
+const ThankYouDetail = ( {
+	title,
+	description,
+	buttonText,
+	href,
+	clickEventName,
+}: ThankYouDetailProps ) => {
+	const handleClick = () => {
```

Comment:
Yes we can certainly do that! I was on the fence about this, thinking it's more convenient to pass a simple string. Although I did later find a scenario where we https://github.com/Automattic/wp-calypso/pull/86204/files#diff-3418dd91735c13487abbc890f2f87b383c1e059fef54d611c9fab18bf13d5f30 instead. I'll update this PR so we pass a function instead of a track event name!